### PR TITLE
fix(natds-themes): change spectrum colors

### PR DESCRIPTION
**Pull request by arielwb**
affects: @naturacosmeticos/natds-themes

DSY-2216

# Description

This PR changes the spectrum colors, adding more shades to the main brand colors.
Also, the gray and red colors from the Aesop spectrum were changed.

### CODE REVIEW

File: packages/natds-themes/bin/colors-to-code-output.json [hunk 1]
```diff
@@ -0,0 +1,3796 @@
+{
```
Comment (yasminnvaz) [inline packages/natds-themes/bin/colors-to-code-output.json:1]: Esse arquivo vem de onde?

File: packages/natds-themes/build/android/colors.xml [hunk 2]
```diff
@@ -20,19 +20,26 @@
  */
 -->
 <resources>
```
Comment (yasminnvaz) [inline packages/natds-themes/build/android/colors.xml:22]: Essas cores que entraram, são do spectrum? Não vi outro lugar mudando assim 🤔

Comment (arielwb) [reply packages/natds-themes/build/android/colors.xml:22]: boaaaaa, deixamos passar isso desde o PR anterio. Good eye! ;)

File: packages/natds-themes/bin/colors-to-code-output.json (hunk 1 above)
Comment (arielwb) [reply packages/natds-themes/bin/colors-to-code-output.json:1]: de um plugin do figma chamado colors-to-code
https://www.figma.com/community/plugin/857997781988109369

**Pull request merged**